### DISCUSSION TITLE
Reload a new rom on the fly

### DISCUSSION THREAD

**Discussion by shaan1974** [EmulatorJS/EmulatorJS#715] [General]
Hello,

I saw that it's possible to reset the running emulator using :

window.EJS_emulator.gameManager.functions.restart()

Is it possible to reload a new rom directly inside the instance of the emulator ?

Thanks

**Comment by ethanaobrien** [comment]
Theres no way to terminate and load a new rom, because theres no true way to terminate and destroy webassembly. An iframe is an easy way to do what youre looking for

**Reply by shaan1974** [reply]
Hello,

I'v found a solution using a tag object with type "text/html", it's like an iframe except that you don't play with the attribute "src" but with attribute "data".

Using this method it's allow me to change the content of the object tag and load a new game.

By the way with this method the fullsize button is not working any more ( i don't know with iframe ).

But it's possible to overwritte the "toggleFullScreen" function of "EJS_emulator" to do some trick to get the fullscreen.

By the way great job with this tool.

**Reply by ethanaobrien** [reply]
> By the way with this method the fullsize button is not working any more ( i don't know with iframe ).

When using iframes, you need to allow the iframe to use fullscreen 

https://www.allowfullscreen.com/

**Reply by shaan1974** [reply]
Thanks for the tip, i was not aware of this attribute for iframe.
